ruby_connectors/spec/github_connector/cataloger_spec.rb

```
@@ -0,0 +1,22 @@
+require 'spec_helper'
+require 'ruby_connectors/github_connector/cataloger'
+
+RSpec.describe RubyConnectors::GithubConnector::Cataloger do
+  let(:cataloger) { described_class.new }
+
+  describe '#catalog' do
```

rubocop

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)